source/css/sass/mixins/_typography.scss

```diff
@@ -109,3 +109,17 @@
 @mixin label-tag-typography() {
   @include _label-typography($color-primary-normal);
 }
+
+@mixin hidden-link($hover-color: $color-primary-dark) {
```

Comment:
Not sure about `hidden` here, as that usually implies that the thing isn't available (e.g. `aria-hidden`), rather than it's there but disguised. `camouflaged-link` perhaps? Not entirely happy with that either but it's closer in meaning. Any ideas?

Reply:
As it's a(n unfortunately) common pattern, let's look elsewhere for inspiration.

Reply:
Bootstrap has `text-reset`, but only applies to colours. Couldn't see any others.

Wonder about `covert-link`? 👀

Reply:
Wondering about a slightly different approach. Instead of putting everything for the link into the mixin, instead the mixin could just be concerned with inheritance. Then we could have something like:

```scss
@mixin inherit-all-with-priority($priorities: ()) {
  @each $priority in $priorities {
    #{$priority}: inherit;
  }
  all: inherit;
}


.some-link {
  @include inherit-all-with-priority(color, text-decoration);
  @supports (all: inherit) {
      cursor: pointer;
  }

  &:hover {
    color: $hover-color;
  }
}

```

Reply:
Could work. `color` and `text-decoration` are fallbacks for `all`; could make the override part of it too (for the lack of `all-except`x). Leaves the hover link specific, probably useful as a mixin still?

Reply:
Tackled in 9aebd1d.